src/css/main.less

```diff
@@ -69,7 +69,12 @@ a:focus {
 
 .btn {
   transition: all 0.2s ease-in-out;
+}
+
+.btn.btn-lg {
   padding: @base-font-size*0.5 @base-spacing-unit*2;
+  font-size: @base-font-size;
+  border-radius: @base-spacing-unit*0.5;
 }
```

Comment:
Can you please move the `.btn-lg` defintion to `L#524` where all the rest remaining ones are? The `.btn` at line `74` should really only contain the base definition and imho shouldn't even be there at all..